README.rst

```diff
@@ -91,6 +91,36 @@ If necessary, override default values for authentication policy:
 
     # multiauth.policy.fxa.realm = Realm
 
+Handling multiple FxA clients
+'''''''''''''''''''''''''''''
+
+If you want to isolate data between two FxA apps using the same Kinto
+service to sync their data you can define client specific
+configuration:
+
+::
+   
+    fxa-oauth.clients.notes.client_id = 89513028159972bc
+    fxa-oauth.clients.notes.required_scope = profile app-notes
+
+    fxa-oauth.clients.todo.client_id = 1805184631529d5a
+    fxa-oauth.clients.todo.required_scope = profile app-todo
+
+
+In that case Kinto will give you two different `user_id`:
```

Comment:
Instead of `in that case`, I would say something like `Depending on the requested scopes, Kinto Fxa will assign a user id or another (using suffix)`